static/data/2017/9/events.json

```diff
@@ -53,4 +53,13 @@
     "website"  : "http://reactalicante.es/",
     "logo"     : "https://pbs.twimg.com/profile_images/906151350588887040/XC5RD0t-_400x400.jpg"
   }
+  {
+    "name" : "PyCon",
+    "start": "04/09/2017",
+    "end"  : "05/09/2017",
+    "location" : "Delhi, India",
+    "type" : "confrence",
```

Comment:
@Ankk98 Seems like you made a typo here.

Reply:
resolved